atlassian_jwt_auth/contrib/django/middleware.py

```diff
@@ -2,7 +2,8 @@
 from django.utils.deprecation import MiddlewareMixin
 
 from atlassian_jwt_auth.frameworks.django.middleware import (
-    OldStyleASAPMiddleware
+    OldStyleASAPMiddleware,
+    asap_middleware as ASAPMiddleware,
```

Comment:
I'm not a fan of this particular change because Django tends to follow the normal Python convention of `ClassName` and `function_name` when distinguishing between the old-style and new-styles middleware.

I don't mind importing `asap_middleware` but I'd rather not call it `ASAPMiddleware` since I think that will confuse callers.